pygmt/clib/session.py

```diff
@@ -28,25 +28,38 @@
 from pygmt.helpers import data_kind, dummy_context, fmt_docstring, tempfile_from_geojson
 
 FAMILIES = [
```

Comment:
Added `GMT_IS_IMAGE`, `GMT_IS_POSTSCRIPT` and `GMT_IS_CUBE` although they're not used now, but they will be used in the future.